### PR TITLE
[122] Implement report UI

### PR DESCRIPTION
**Pull request by pisuttun**
## What did you do

- add job's report page
- add reporterName and jobTitle in Report dtos
image

## Demo
- https://dev.modela.miello.dev/job/1/reports //logged in as admin 

## Note
- will write unit test after connect to API

### CODE REVIEW

**Comment by PKhing** [comment]
image
Clicking on job name should redirect to job detail page

File: apps/web/src/modules/job/report/components/JobReportCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <Typography variant="h5" textAlign="center">
+        รายละเอียดปัญหา
+      </Typography>
+      <Divider />
+      <Typography variant="body1">
+        งานที่ได้รับการแจ้งปัญหา: {jobTitle}
+      </Typography>
+      <Typography variant="body1">มีการแจ้งปัญหา 2 ครั้ง</Typography>
+      {reports.map((report, idx) => (
+        <ReportDetail key={idx} {...report} />
```
Comment (miello) [inline apps/web/src/modules/job/report/components/JobReportCard/index.tsx:26]: Maybe use `reportId` instead

File: apps/web/src/modules/job/report/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { UserType } from '@modela/database'
+import { GetReportsDto } from '@modela/dtos'
```
Comment (miello) [inline apps/web/src/modules/job/report/pages/index.tsx:2]:
```suggestion
import { UserType, GetReportsDto } from '@modela/dtos'
```

**Review by miello — CHANGES_REQUESTED** [review]
And please include Task ID to PR title also

**Review by miello — APPROVED** [review]

**Pull request merged**